ui/app/templates/allocations/allocation/task/fs.hbs

```diff
@@ -1 +1,70 @@
-{{outlet}}
\ No newline at end of file
+{{task-subnav task=task}}
+<section class="section closer">
+  {{#if task.isRunning}}
+    {{! template-lint-disable table-groups }}
+    <table class="fs-explorer table is-compact">
+      <thead>
+        <tr>
+          <td colspan="3">
+            <nav class="breadcrumb fs" data-test-fs-breadcrumbs>
+              <ul>
+                <li class={{if breadcrumbs "" "is-active"}}>
+                  {{#link-to "allocations.allocation.task.fs-root" task.allocation task activeClass="is-active"}}
+                    {{task.name}}
+                  {{/link-to}}
+                </li>
+                {{#each breadcrumbs as |breadcrumb|}}
+                  <li class={{if breadcrumb.isLast "is-active"}}>
+                    {{#link-to "allocations.allocation.task.fs" task.allocation task breadcrumb.path activeClass="is-active"}}
+                      {{breadcrumb.name}}
+                    {{/link-to}}
+                  </li>
+                {{/each}}
+              </ul>
+            </nav>
+          </td>
+        </tr>
+      </thead>
+      {{#if isFile}}
+        <tbody>
+          <tr>
+            <td>
+              <div data-test-file-viewer>placeholder file viewer</div>
```

Comment:
Hm. Although there's nothing wrong with this, I wasn't imagining the file viewer going into a table cell.

I was expecting something like a `.boxed-section` with the breadcrumbs going in the `.boxed-section-head` and a table for the directory listing in the `.boxed-section-body`.

What's the argument for making the whole thing a table?

Reply:
I definitely wasn’t meaning to force any particular markup on the file viewing, I just chose this because it was able to handle the table spanning the whole container, which is how I started and then just kept with due to inertia. But I’ve changed it to use `boxed-section` etc, thanks for the pointer!